src/components/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from "react";
+
+import { classNames } from "~/utils";
+
+import { Box } from "../Box";
+import { button, ButtonVariants } from "./Button.css";
+
+export type ButtonProps = ButtonVariants & {
+  children: ReactNode;
+  disabled?: boolean;
+  className?: string;
```

Comment:
**suggestion**: We need more props here

I'm thinking about at least `type` and `onClick`

Reply:
Please add those props - we will need them as soon as possible when we release next version 😄

Reply:
The button is now `forwardRef<HTMLButtonElement, ButtonProps>` component which means it accepts all native HTML tags, including `type` and `onClick`